src/main/java/org/broadinstitute/dsm/jobs/UPSTrackingJob.java

```diff
@@ -30,10 +29,12 @@
 public class UPSTrackingJob implements Job {
 
     private static final Logger logger = LoggerFactory.getLogger(UPSTrackingJob.class);
-            private static final String SQL_SELECT_KITS = "SELECT * FROM ddp_kit kit LEFT JOIN ddp_kit_request req " +
-                    " ON (kit.dsm_kit_request_id = req.dsm_kit_request_id) WHERE req.ddp_instance_id = 9 and kit_label not like \"%\\\\_1\"  and (ups_tracking_status not like \"D %\" or ups_return_status not like \"D %\")" +
+            private static final String SQL_SELECT_KITS = "SELECT kit.dsm_kit_request_id, kit.kit_label, kit.tracking_to_id, kit.tracking_return_id, kit.error, kit.message, kit.receive_date, kit.ups_tracking_status, kit.ups_tracking_date, kit.ups_return_date, kit.ups_return_status, req.bsp_collaborator_participant_id" +
+                    " req.external_order_number, kit.CE_order FROM ddp_kit kit LEFT JOIN ddp_kit_request req " +
+                    " ON (kit.dsm_kit_request_id = req.dsm_kit_request_id) WHERE req.ddp_instance_id = 9 and kit_label not like \"%\\\\_1\"  and (ups_tracking_status not like \"? %\" or ups_return_status not like \"? %\")" +
```

Comment:
Hardcoding an internal id may make this unusable in non-prod environments.  Probably safer to use a more stable identifier like the instance name.

Reply:
OMG :doh: I didn't see this, but removed now